src/tasks.js

```diff
@@ -0,0 +1,103 @@
+ko.tasks = (function () {
```

Comment:
I wouldn't call either of these "substantial", and the code is not copied exactly (I've made minor changes). Still, I think it makes sense to include more detail about the license. So how is this?

```
// From https://github.com/petkaantonov/bluebird * Copyright (c) 2014 Petka Antonov * License: MIT
```